src/SprykerEco/Yves/Unzer/Theme/default/views/credit-card/credit-card.twig

```diff
@@ -0,0 +1,14 @@
+{% extends model('template') %}
+
```

Comment:
Please create a template for both views.

Reply:
I don't see the reason, what should be in those templates?